public/js/lib/highlight/prism.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-env browser, jquery */
+
+/**
+ * WHY
+ * ---
+ * This extends the Prism.js module to provide some useful and needed functions
+ * for better integration with CodiMD.
+ * It also allows central provisioning of additional languages.
+ */
+
+require('prismjs/themes/prism.css')
+require('prismjs/components/prism-wiki')
+require('prismjs/components/prism-haskell')
+require('prismjs/components/prism-go')
+require('prismjs/components/prism-rust')
+require('prismjs/components/prism-typescript')
+require('prismjs/components/prism-jsx')
+require('prismjs/components/prism-makefile')
+require('prismjs/components/prism-gherkin')
+
+import Prism from 'prismjs'
+
+const languages = ['haskell', 'go', 'rust', 'typescript', 'typescript', 'js', 'jsx', 'gherkin', 'tiddlywiki', 'mediawiki', 'wiki', 'cmake']
+
+Prism.listLanguages = function () {
```

Comment:
We extend a foreign library? This seems counter-intuitive? And also I don't think it's needed?